src/modules/did/did.js

```diff
@@ -49,7 +51,7 @@ class DIDModule {
    * @returns {*}
    */
   createNewOffchainTx(did, didDocRef) {
-    const hexId = getHexIdentifierFromDID(did);
+    const hexId = typedHexDID(this.api, did).asDid;
```

Comment:
You know this did won't be a `did:key` so its more efficient to use `getHexIdentifier(did, DockDIDQualifier, DockDIDByteSize)` here than calling `typedHexDID` which creates an unnecessary object

Reply:
I don't think it will differ in efficiency despite an unnecessary object will be created.